frontends/p4/def_use.cpp

```diff
@@ -817,8 +817,8 @@ bool ComputeWriteSet::preorder(const IR::AssignmentStatement* statement) {
 bool ComputeWriteSet::preorder(const IR::SwitchStatement* statement) {
     LOG3("CWS Visiting " << dbp(statement));
     visit(statement->expression);
-    auto locs = get(statement->expression);
-    auto defs = currentDefinitions->writes(getProgramPoint(), locs);
+    auto locs = getWrites(statement->expression);
+    auto defs = currentDefinitions->writes(getProgramPoint(statement->expression), locs);
```

Comment:
What about this change?  It is not (just) a renaming -- there's a real change here.  Is it intentional?